### ISSUE TITLE
[MNG-3003] maven-plugin-testing-tools: Updates for PLXUTILS-31 patches

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#4906]
**[Barrie Treloar](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=baerrach)** opened **[MNG-3003](https://issues.apache.org/jira/browse/MNG-3003?redirect=false)** and commented

Bumping of versions in dependencies to fix PLXUTILS-31

---
No further details from [MNG-3003](https://issues.apache.org/jira/browse/MNG-3003?redirect=false)

**Comment by jira-importer** [comment]
**[Barrie Treloar](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=baerrach)** commented

Upgrade of

```
    <dependency>
      <groupId>org.codehaus.plexus</groupId>
      <artifactId>plexus-utils</artifactId>
      <version>1.4.2</version>
    </dependency>
and
    <dependency>
      <groupId>org.apache.maven.shared</groupId>
      <artifactId>maven-test-tools</artifactId>
      <version>1.0-alpha-2-SNAPSHOT</version>
      <scope>test</scope>
```

**Comment by jira-importer** [comment]
**[Barrie Treloar](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=baerrach)** commented

Fixed in maven-plugin-testing-tools:1.0-alpha-2-SNAPSHOT
